domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt

```diff
@@ -72,7 +72,7 @@ object InteractionObjectTestBuilder {
       }.build()
   }.build()
 
-  fun createWholeNumber(isNegative: Boolean, value: Int): InteractionObject {
+  fun createWholeNumber(isNegative: Boolean = false, value: Int): InteractionObject {
     // Whole number fractions imply '0/1' fractional parts.
```

Comment:
We'll probably want to remove the parameter `isNegative` from this method, as there can't be a negative whole number.
Adding @vinitamurthi for confirmation.

Reply:
Yes @prayutsu sir